code/datums/mutations/body.dm

```diff
@@ -230,6 +231,108 @@
 		return
 	owner.physiology.burn_mod *= 2
 
+/datum/mutation/human/badblink
+	name = "Spatial Instability"
+	desc = "Subject has a very weak link to spatial reality, and may be displaced. Often causes extreme nausea."
+	quality = NEGATIVE
+	text_gain_indication = "<span class='warning'>The space around you twists sickeningly.</span>"
+	text_lose_indication = "<span class'notice'>The space around you settles back to normal.</span>"
+	difficulty = 18//high so it's hard to unlock and abuse
+	instability = 10
+	var/warpchance = 0
+
+/datum/mutation/human/badblink/on_life()
+	if(prob(warpchance))
+		var/warpmessage = pick(
+		"<span class='warning'>With a sickening 720 degree twist of their back, [owner] vanishes into thin air.</span>",
+		"<span class='warning'>[owner] does some sort of strange backflip into another dimension. It looks pretty painful.</span>",
+		"<span class='warning'>[owner] does a jump to the left, a step to the right, and warps out of reality.</span>",
+		"<span class='warning'>[owner]'s torso starts folding inside out until it vanishes from reality, taking [owner] with it.</span>",
+		"<span class='warning'>One moment, you see [owner]. The next, [owner] is gone.</span>")
+		owner.visible_message(warpmessage, "<span class='userdanger'>You feel a wave of nausea as you fall through reality!</span>")
+		var/warpdistance = rand(10,15)
+		do_teleport(owner, get_turf(owner), warpdistance, channel = TELEPORT_CHANNEL_FREE)
+		owner.adjust_disgust((warpchance * warpdistance))
+		warpchance = 0
+		owner.visible_message("<span class='danger'>[owner] appears out of nowhere!</span>")
+	else
+		warpchance += 0.25
+
+/datum/mutation/human/acidflesh
+	name = "Acidic Flesh"
+	desc = "Subject has acidic chemicals building up underneath their skin. This is often lethal."
+	quality = NEGATIVE
+	text_gain_indication = "<span class='userdanger'>A horrible burning sensation envelops you as your flesh turns to acid!</span>"
+	text_lose_indication = "<span class'notice'>A feeling of relief covers you as your flesh goes back to normal.</span>"
+	difficulty = 18//high so it's hard to unlock and use on others
+
+/datum/mutation/human/acidflesh/on_life()
+	if(prob(25))
+		to_chat(owner, "<span class='danger'>Your acid flesh bubbles...</span>")
```

Comment:
does this not spam chat?

Reply:
Intended, presses urgency and it squashes anyways. Not as frequent as you might expect either.

Reply:
It doesn't squash when you have a bunch of players chatting in radio or doing literally anything else you memer.

It needs a cooldown or something to ensure it RNG won't clusterheck your chat panel.

Reply:
Ok, I put a 20 second minimum cooldown